CONTRIBUTING.md

```diff
@@ -15,26 +15,28 @@ Numberscope.
 ## If you are experienced, follow these steps...
```

Comment:
the video playlist doesn't exist

Reply:
Oh, I had no idea. I've never bothered to click on the link, since video tutorials are basically never of interest to me... I will just remove that section.